api/index.md

```diff
@@ -33,4 +32,35 @@ The endpoints can be roughly grouped into four categories:
     graph query language [SPARQL](https://www.w3.org/TR/rdf-sparql-query/){: target="_blank"}. This is useful for complex node connections which would require multiple API calls; for example, "hate crimes motivated by disability status in Californian cities".
 
 -   **Utilities**: These are Python notebook-specific APIs for helping with
-    Pandas DataFrames, etc.
\ No newline at end of file
+    Pandas DataFrames, etc.
+
+In addition, Data Commons provides additional tools for accessing its data that call the REST APIs under the hood:
+
+- [Google Sheets](sheets/index.md): provides several custom functions that populate spreadsheets with data from the Data Commons knowledge graph
+- [Web Components](web_components/index.md): provides JavaScript APIs and HTML templates that allow you to embed Data Commons data and visualizations into web pages
+
+
+## API keys {: #get-key}
+
+A key is required by some APIs to authenticate and authorize requests. 
+- All REST [V2](rest/v2/index.md) and [V1](rest/v1/index.md) APIs. These requests are served by endpoints at `api.datacommons.org`.
+- All requests coming from a custom Data Commons instance. These are also served by `api.datacommons.org`.
+- All DataGemma APIs <!-- TODO: Add link to docs about these when available -->. These requests are served by endpoints at `datagemma.datacommons.org`.
```

Comment:
"All DataGemma APIs" -> "Data Commons NL API requests" to be consistent with the API product name in the portal.

Reply:
SG. Changed.